stylesheets/_component.forms.scss

```diff
@@ -756,6 +756,12 @@ fieldset.form__group {
 }
 
 .form__group--compound {
+    > legend.control__label {
```

Comment:
Why not just `> .control__label`? this shouldn't target the control labels inside `controls` if that was the concern?

Reply:
Valid point, removed 🔥